docs/rules/prefer-spread.md

```diff
@@ -0,0 +1,51 @@
+# Suggest using of the spread operator instead of `.apply()` callings. (prefer-spread)
+
+Up to now, we had been using `Function.prototype.apply()` callings for the variadic arguments.
```

Comment:
I think term `varidic arguments` only applies to `...` operator in C++. The more common term is `varidic function`. Please correct me if I'm wrong.